v2/apiserver/internal/system/rest/system_endpoints.go

```diff
@@ -77,6 +83,24 @@ func (h *SystemEndpoints) healthz(w http.ResponseWriter, r *http.Request) {
 	)
 }
 
+// unversionedPing returns the api server version and http.StatusOK.
+// This is handy for auxiliary components to verify their connectivity.
```

Comment:
Should we mention that this exists mainly for the benefit of clients that may have no clue what version of the API they're talking to and might like to find out?